mod/metrics/set/type/category_value.rb

```diff
@@ -38,7 +42,7 @@ def editor
   end
 
   def pretty_value
-    card.pretty_values.join JOINT
+    card.pretty_values.join ", "
```

Comment:
I think everywhere else that uses this joint is about making a string for the lookup table.  This is for presentation.  Potentially two different joints soon...